CHANGELOG.md

```diff
@@ -4,6 +4,48 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 
 ## [Unreleased]
 
+## [5.0.0]
+
+### Added
+- Support for Java 17
+- Support for Groovy 2.5
+- Use jitpack for verifying artifacts
+- Support to configure details on HTTP client max connections and connection eviction via environment variables
+- Support collecting metrics on Request class HTTP clients 
+- Support for Protocol Buffer File as a format for exporting data
+- Support for TopoJSON as a format for exporting transformed data
+- Support to get list of Shapefile names
+- Support for XPlanGML 5.4 and XPlanGML 6.0 schema preset
+- Support to allow excluding root properties from structural Retype
+- Support to automatically add the codespace for INSPIRE feature types
+- Support for `xs:all` to the XML Schema Reader
+- Support to export data in Json format. GeoJson format to follow RFC7946 SPEC
```

Comment:
Wasn't this present before already?

Reply:
Yes, I added this thinking to highlight RFC7946 SPEC because I guess before exported Json or GeoJson were not validated properly. 
I will remove this line it will confuse the readers.